### ISSUE TITLE
T-Clock doesn't work at all on Windows 11 build 22000.51

### ISSUE THREAD

**Issue by Skyyblaze** [White-Tiger/T-Clock#256]
Since the clock in the Taskbar was redesigned, T-Clock currently doesn't work on Windows 11 build 22000.51. This was expected seeing how the clock is now integrated in the action-center widget but I would appreciate if this could be looked into.

**Comment by iJahangard** [comment]
Most part of win11 app now are UWP apps
Also search start ans taskbar which is completely redesigned I guess

**Comment by sagars007** [comment]
Yeah as per Windows 11 changelog: applications will be unable to modify areas of the taskbar

**Comment by blackcrack** [comment]
make a branch and develops it further, White-Tiger does it not...
Did you see the last code-commit ?

> White-Tiger build: fixed sign-n-zip.bat (broken by 50bf612); LOGGING environmenta…
 ... :heavy_check_mark: 2eb7e2a on 8 Jun 2018   :clock4: 531 commits     


the last Binarys and the last Sourcecode from the Orginal:  
https://www.donationcoder.com/forum/index.php?topic=21944.0
> Author | Stoic Joker - Has officially closed the project as of Saturday March 10, 2012

[<h2>v2.4.4 release candidate</h2>](https://github.com/White-Tiger/T-Clock/releases)
>White-Tiger released this on 28 Jan 2018   

best

**Comment by GalaxySH** [comment]
Per the taskbar section of the [deprecations and removals log](https://www.microsoft.com/en-us/windows/windows-11-specifications#primaryR4), it seems that T-Clock is officially dead.

> Apps can no longer customize areas of the Taskbar.

(this was already mentioned, I wanted to provide a link)

**Comment by blackcrack** [comment]
@White-Tiger ,
Hi Renè, 
hab dein Thumps up gesehen, was iss nu, programmierst Du hier nu weiter oder nich ?
liebe Grüße
Blacky

**Comment by adacom81** [comment]
same as others TClock did not work in windows 11 - i also use Startisback - there is a pre release beta - i added that and it made changes to the right side of the taskbar - i then tried TClock again and it works perfectly

**Comment by Eriond** [comment]
Yep, StartIsBack is a viable workaround for now.

**Comment by ThisIsTrue** [comment]
Being a terrible programmer at best, I don't know if this is easy or hard, but is it possible to redo TClock as a stand-alone app (like a gadget or widget) that we can place "anywhere" on a screen, and have the ability to click on the clock to bring up the calendar and such as per normal? Which is to say, leave the taskbar entirely so it would work regardless. Then we wouldn't have to fuss with all these workarounds, which bring their own problems.

**Comment by flacy1** [comment]
Hello,
how long wait alarms for any days of the years at **t-clock redux x64**?
